discovery-provider/src/queries/get_health.py

```diff
@@ -251,3 +249,22 @@ def get_latest_chain_block_set_if_nx(redis=None, web3=None):
             logger.error(f"Could not set values in redis for get_latest_chain_block_set_if_nx: {e}")
 
     return latest_block_num, latest_block_hash
+
+# returns a tuple (<boolean>, <boolean>) that represents whether
+# the given blockhash is present, the given blocknumber is passed
+def get_block_confirmation(blockhash, blocknumber):
```

Comment:
This is gonna be much less error prone if it returns a dictionary - returning two unlabeled bools is asking for trouble 😉

Reply:
for sure I like it! thanks